cli/version/version.go

```diff
@@ -27,9 +27,14 @@ Usage of version:
 	cfssl version
 `
 
+// Returns the formatted version string.
```

Comment:
golint error: the exported function comments/docs should start with the function name.